packages/terra-date-picker/src/CalendarFilter.module.scss

```diff
@@ -0,0 +1,13 @@
+:local {
+  .calendar-filter {
+    border: var(--terra-date-picker-calendar-filter-border, 1px solid #aeaeae);
+    width: var(--terra-date-picker-calendar-filter-width, 16.140642rem);
+    
+    // To keep month navigation button height consistent across viewports.
```

Comment:
Does this issue also occur in date-picker in small and large breakpoints?

Reply:
yes in date-picker also we can see this issue. when it is rendered within portal button grows bigger than when it is rendered without portal.

without portal : 
<img width="273" alt="Screen Shot 2020-06-03 at 2 38 45 PM" src="https://user-images.githubusercontent.com/12869809/83618625-578a6d80-a5a8-11ea-8f36-aee83da04e6a.png">

with portal : 
<img width="265" alt="Screen Shot 2020-06-03 at 2 39 35 PM" src="https://user-images.githubusercontent.com/12869809/83618648-5f4a1200-a5a8-11ea-9813-2b5db51e939b.png">

Reply:
Should these styles be added to [react_datepicker.module.scss](https://github.com/cerner/terra-framework/blob/67d57a6dd5e450f637c74892bd614eef36d08d77/packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss) so that it also fixes it for popup and portal picker and not just calendar filter?

Reply:
but I think this was an intentional change added in [react_datepicker.module.scss](https://github.com/cerner/terra-framework/blob/67d57a6dd5e450f637c74892bd614eef36d08d77/packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss#L493-L499) in [PR 866](https://github.com/cerner/terra-framework/pull/866/files#diff-51e1bb4a0aafbfca4d959f9018880c8aR481-R487) to adjust the size based on whether it is in the portal or popup mode. 

since calendar-filter will not be having portal mode keeping these change limited for calendar-filter.

Reply:
Yes, you are correct so these styles should stay here.